server/controllers/medical/patients/documents.js

```diff
@@ -68,21 +68,21 @@ function create(req, res, next) {
   });
 
   db.exec(sql, [records])
-  .then(() => {
-    // publish a patient update event
-    Topic.publish(Topic.channels.MEDICAL, {
-      event : Topic.events.UPDATE,
-      entity : Topic.entities.PATIENT,
-      user_id : req.session.user.id,
-      id : req.params.uuid,
-    });
-
-    res.status(201).json({
-      uuids : records.map(row => uuid.unparse(row[0])),
-    });
-  })
-  .catch(next)
-  .done();
+    .then(() => {
+      // publish a patient update event
+      Topic.publish(Topic.channels.MEDICAL, {
+        event : Topic.events.UPDATE,
+        entity : Topic.entities.PATIENT,
+        user_id : req.session.user.id,
+        id : req.params.uuid,
+      });
+
+      res.status(201).json({
+        uuids : req.files.map(file => file.filename),
```

Comment:
it is not obvious that `filename` is a uuid, a comment on that will be not bad.

Ex. 
```js
// document file name is in the string uuid format
// return a set of these files names as an array of uuid
```